test_communication/test/test_messages_c.cpp

```diff
@@ -129,25 +129,25 @@ class CLASSNAME (TestMessagesFixture, RMW_IMPLEMENTATION) : public ::testing::Te
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string_safe();
     EXPECT_EQ(0, strcmp(rcl_subscription_get_topic_name(&subscription), topic_name));
 
-    rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
+    rcl_wait_set_t wait_set_connection = rcl_get_zero_initialized_wait_set();
```

Comment:
Why adding `_connection` to this variable? How is the wait set related to a "connection"?

Instead the block could be wrapped in a local scope to avoid the colliding variable name.